### PR TITLE
Adding setup scripts, Dockerfile, new README, and tutorial

### PR DESCRIPTION
**Pull request by jtpaasch**
Added setup scripts, a Dockerfile, a new README, and the tutorial.

### CODE REVIEW

File: vibes-tools/docs/tutorial/Tutorial.md [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+The `vibes-init` tool will scaffold a patching project. 
+
+To begin, run `vibes-init`:
+
+```
+vibes-init \
+  --patch-names=my-patch \
+  --binary=main.exe \
+  --patched-binary=main.patched.exe \
+  --target=bap:arm \
```
Comment (bmourad01) [inline vibes-tools/docs/tutorial/Tutorial.md:109]: Same here.

File: vibes-tools/docs/tutorial/Tutorial.md [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+      "storage-class": ["register", {"at-entry": "R0", "at-exit": "R0"}]
+    }
+  ]
+}
+```
+
+This tells VIBES the following things:
+
+* `patch-point` start patching at address `0x103d4:32` (the `32` indicates that the address is a 32-bit number)
+* `patch-size`: overwrite 8 bytes
```
Comment (bmourad01) [inline vibes-tools/docs/tutorial/Tutorial.md:198]: Should say 4 bytes here.

File: vibes-tools/README.md [hunk 3]
```diff
@@ -1,152 +1,143 @@
...
-            |   |-- ...
-            |-- "bin/"
-                |
-                |-- "dune" (defines a "vibes-foo" executable)
-                |-- "main.ml" (defines the Cmdliner CLI)
+vibes-init \
+  --patch-name=my-patch \
+  --binary=program.exe \
+  --patched-binary=program.patched.exe \
+  --target=bap:arm \
```
[the remark this answers is not in the document]
Comment (jtpaasch) [reply vibes-tools/README.md:111]: 👍

File: vibes-tools/docs/tutorial/Tutorial.md (hunk 1 above)
Comment (jtpaasch) [reply vibes-tools/docs/tutorial/Tutorial.md:109]: 👍

File: vibes-tools/docs/tutorial/Tutorial.md (hunk 2 above)
Comment (jtpaasch) [reply vibes-tools/docs/tutorial/Tutorial.md:198]: Good catch!

**Comment by jtpaasch** [comment]
Okay, changes made.

**Review by bmourad01 — APPROVED** [review]

**Pull request merged**